docs/building-a-form/common-definitions.md

```diff
@@ -65,91 +65,121 @@ A common type-ahead widget that lets a user type in values and narrow down a lon
 
 ### Current or past dates
 
-The common date field with current or past validation set.
+The common date field with current or past validation set (i.e., dates in the future are not valid). Call this exported function and pass it the label to be used on the field.
 
-- File: [/src/js/definitions/currentOrPastDate.js](../../src/js/definitions/currentOrPastDate.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+```js
+import currentOrPastDateUI from 'us-forms-system/lib/js/definitions/currentOrPastDate';
+
+uiSchema: {
+  birthdate: currentOrPastDate('Date of Birth')
+}
+```
+Source: [/src/js/definitions/currentOrPastDate.js](../../src/js/definitions/currentOrPastDate.js)
 
 ### Current or past month/year
 
-The common date field without the day field and with current or past validation set.
+The common date field without the day field and with current or past validation set (i.e., dates in the future are not valid). Call this exported function and pass it the label to be used on the field.
 
-- File: [/src/js/definitions/currentOrPastMonthYear.js](../../src/js/definitions/currentOrPastMonthYear.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+```js
+import currentOrPastMonthYear from 'us-forms-system/lib/js/definitions/currentOrPastMonthYear';
+
+uiSchema: {
+  lastContact: currentOrPastMonthYear('Last Contact')
+}
+```
+Source:  [/src/js/definitions/currentOrPastMonthYear.js](../../src/js/definitions/currentOrPastMonthYear.js)
 
 ### Date
 
-The common date field with basic date validation.
+The common date field with basic date validation. Call this exported function and pass it the label to be used on the field.
+```js
+import dateUI from 'us-forms-system/lib/js/definitions/date';
 
-- File: [/src/js/definitions/date.js](../../src/js/definitions/date.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+uiSchema: {
+  startDate: dateUI('startDate')
+}
+```
+Source: [/src/js/definitions/date.js](../../src/js/definitions/date.js)
 
 ### Date range
 
-Two common date fields with validation to ensure they form a valid range.
+Two common date fields with validation to ensure they form a valid range. Call this exported function.
+```js
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
 
-- File: [/src/js/definitions/dateRange.js](../../src/js/definitions/dateRange.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+uiSchema: {
+  servicePeriod: dateRangeUI('servicePeriod')
+}
+```
+Source: [/src/js/definitions/dateRange.js](../../src/js/definitions/dateRange.js)
 
 ### File upload
 
-The file upload field. Requires a specified endpoint for the upload.
+*Note: This is currently not functional. The file upload field requires a specified endpoint for the upload. Documentation will be updated later.*
 
-- File: [/src/js/definitions/file.js](../../src/js/definitions/file.js)
-- `uiSchema`: Yes
-- `schema`: Yes (the same as in vets-json-schema)
```

Comment:
There's a `schema` in file.js but it looks like nobody uses it and it's just there as a "serving suggestion".

Reply:
Also, file.js is strange because it does an `export default` on the uiSchema but just plain `export` on the schema. So to import both parts you'd have to do something like:
```js
import { default as fileUI, fileSchema } from '/path/to/file';
```